editor/src/clj/editor/outline_view.clj

```diff
@@ -488,7 +488,6 @@
         image-view-icon (ImageView.)
         visibility-button (doto (ToggleButton.)
                             (ui/add-style! "visibility-toggle")
-                            (.addEventFilter MouseEvent/MOUSE_PRESSED ui/ignore-event-filter)
```

Comment:
Clicking on the visibility toggle was somehow triggering this issue, and I was also trying to avoid opening the resource on double click. The latter was fixed [here](https://github.com/defold/defold/pull/10183/files#r1948029163), so this is not required.